tlsutil/config.go

```diff
@@ -561,6 +569,12 @@ func (c *Configurator) VerifyServerHostname() bool {
 	return c.base.VerifyServerHostname || c.autoEncrypt.verifyServerHostname
 }
 
+// TransportCredentials returns credentials to be used with eg a grpc server.
+func (c *Configurator) TransportCredentials() *credentials.TransportCredentials {
+	creds := credentials.NewTLS(&tls.Config{Certificates: []tls.Certificate{*c.Cert()}})
+	return &creds
```

Comment:
```suggestion
	return creds
```
See comment about `TransportCredentials()` pointer dereference.